### PR TITLE
Add more code to main branch

### PR DESCRIPTION
**Pull request by cabouman**

### CODE REVIEW

File: phantom.py [hunk 1]
```diff
@@ -0,0 +1,244 @@
+import numpy as np
+
+def gen_shepp_logan(num_rows,num_cols):
+    """
+    Generate a Shepp Logan phantom
+
```
Comment (sjkisner) [inline phantom.py:6]: oh this is bad

Comment (cabouman) [reply phantom.py:6]: I think it's fine.

Comment (gbuzzard) [reply phantom.py:6]: I don't think it's that bad.

Comment (sjkisner) [reply phantom.py:6]: ok you're right

File: phantom.py [hunk 2]
```diff
@@ -0,0 +1,244 @@
...
+    Generate a Shepp Logan phantom
+    
+    Args: 
+        num_rows: int, number of rows.
+        num_cols: int, number of cols.
+
+    Return:
+        out_image: 2D array, num_rows*num_cols
+    """
+
```
Comment (sjkisner) [inline phantom.py:14]: oh what's this??

File: phantom.py [hunk 3]
```diff
@@ -0,0 +1,244 @@
...
+    # The function describing the phantom is defined as the sum of 10 ellipses inside a 2×2 square:
+    sl_paras = [
+        {'x0': 0.0, 'y0': 0.0, 'a': 0.69, 'b': 0.92, 'theta': 0, 'gray_level': 2.0},
+        {'x0': 0.0, 'y0': -0.0184, 'a': 0.6624, 'b': 0.874, 'theta': 0, 'gray_level': -0.98},
+        {'x0': 0.22, 'y0': 0.0, 'a': 0.11, 'b': 0.31, 'theta': -18, 'gray_level': -0.02},
+        {'x0': -0.22, 'y0': 0.0, 'a': 0.16, 'b': 0.41, 'theta': 18, 'gray_level': -0.02},
+        {'x0': 0.0, 'y0': 0.35, 'a': 0.21, 'b': 0.25, 'theta': 0, 'gray_level': 0.01},
```
Comment (sjkisner) [inline phantom.py:21]: what about this???

File: phantom.py (hunk 2 above)
Comment (sjkisner) [reply phantom.py:14]: nevermind

File: phantom.py (hunk 3 above)
Comment (sjkisner) [reply phantom.py:21]: nevermind

**Pull request closed without merging**